contracts/src/v0.8/dev/functions/FunctionsBillingRegistry.sol

```diff
@@ -637,18 +645,6 @@ contract FunctionsBillingRegistry is
     return currentsubscriptionId;
   }
 
-  /**
-   * @notice Gets subscription owner.
-   * @param subscriptionId - ID of the subscription
-   * @return owner - owner of the subscription.
-   */
-  function getSubscriptionOwner(uint64 subscriptionId) external view override returns (address owner) {
```

Comment:
Why is this being removed?  Did we confirm we don't rely on this anywhere in the tooling?